package.json

```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "generate:docs": "ts-node scripts/generate-docs.ts",
     "postinstall": "patch-package",
-    "lint": "eslint --config .eslintrc.json --ext ts,js .",
+    "lint": "eslint --ignore-path .gitignore --ext ts,js .",
```

Comment:
no need to have `.eslintignore` file, because `.gitignore` file already contain ignored files and folders